### ISSUE TITLE
alfresco/header/SetTitle and browserTitlePrefix

### ISSUE THREAD

**Issue by aviriel** [Alfresco/Aikau#1112]
I try to remove "Alfresco >>" from the browser title. I added new "page.title" to i18n packages that works for old YUI2 pages, I added
```
{
  name: "alfresco/header/Title",
  config: {
    label: (pageTitle != null) ? pageTitle : getPageTitle(),
    setBrowserTitle: false,
  }
}
```
for Aikau pages.

A question: how to make this changes for Aikau pages in Admin Console? I created the page in the same way as Sites Manager page done. So I have
```
{
  id: "SET_PAGE_TITLE",
  name: "alfresco/header/SetTitle",
  config: {
    title: msg.get("tool.something.label")
  }
}
```

It changes the title, but looks like I cannot say that I don't want to use prefix or change the prefix (https://github.com/Alfresco/Aikau/blob/master/aikau/src/main/resources/alfresco/header/Title.js#L211). Can you extend updatePageTitle() function to support it or give me any workaround for it?

**Comment by draperd** [comment]
At the moment it isn't possible do this because of the way that the code has been written. I can raise a feature to implement this chang - how soon do you need this feature to be available?

**Comment by aviriel** [comment]
It would be wonderful if you can do it, I need it yesterday, so I will be happy if you can do it in the next months.

**Comment by draperd** [comment]
Yes, no problem - it's a small change so I'll try and get it done as soon as possible - but it's more likely to be in the next sprint rather than the current one. So you're probably looking at the release after next (i.e. 1.0.73 in a couple of weeks)

**Comment by draperd** [comment]
I've created https://issues.alfresco.com/jira/browse/AKU-1000 to address this

**Comment by aviriel** [comment]
Thanks a lot! I just understood that maybe I was wrong in my request. I want to set browserTitle, but I don't want to use prefix in it.
I can create a pull request that will implement it, but I'm not sure that I can write unit tests for it, if you need it.

**Comment by draperd** [comment]
I've made some updates that are pending merge for the 1.0.73 release next week. You'll now be able to use `hideBrowserTitlePrefix` and `browserTitlePrefix` as configuration options on the `alfresco/header/SetTitle` widget. This means that you'll have the ability to change and/or hide prefixes.

**Comment by draperd** [comment]
This is now resolved in the 1.0.73 release that is now available
